UserService/UserService/src/main/java/com/nt/user/microservice/service/impl/AddressServiceImpl.java

```diff
@@ -0,0 +1,81 @@
+package com.nt.user.microservice.service.impl;
+
```

Comment:
create a ServiceImpl folder as you have created the other folders
It should not be like that the impl folder is inside the service folder